docs/getting_started_guide/Installation_Options/ISO/Installing_SIMP_From_An_ISO.rst

```diff
@@ -3,91 +3,128 @@
-        instructions to enter your partition scheme. For example, for SIMP for
-        CentOS 7, select the ``INSTALLATION DESTINATION`` menu, enter the
-        selected partitioning, select the ``DONE`` button to finalize your disk
-        selections, and then select the ``Begin Installation`` button on the
-        main GUI page to continue. No further GUI interaction will be required.
+      For example, using SIMP for CentOS 7:
 
-      * If you have chosen to encrypt your disks, your installation seems to
-        be paused, and a messages about increasing entropy appears on the
-        screen, you may want to generate some system entropy by pressing random
-        keys on the keyboard for a bit.  This will speed up the installation.
+        #. Click the ``INSTALLATION DESTINATION`` button
+        #. Configure the desired partitioning
+        #. Click the ``DONE`` button to finalize your disk selections
+        #. Click the ``Begin Installation`` button on the main GUI page to
+           continue.
+
+      No further GUI interaction will be required.
+
+   .. NOTE::
+
+      When applying disk encryption (unless booting with ``simp-nocrypt``), the
+      system may seem to pause and display messages about increasing entropy.
+      You can speed up the installation by pressing random keys on the keyboard
+      for a bit (this will generate additional entropy).
 
 #. When the installation is complete, the system will restart automatically.
 
    .. NOTE::
 
-      When the system boots it will display: ``error on start module sha1 not
+      When the system boots, it may display: ``error on start module sha1 not
       found could not insert sha_256 [...]``. This is expected and is a known
-      issue with FIPS and RedHat, it is safe to ignore.
+      issue.
+
+#. Change the default passwords.
+
+   .. WARNING::
 
-#. Change the ``root`` user password
+      There are default passwords present on the system that should be changed
+      prior to deploying the system.
 
-   a. At the console, log on as ``root`` and type the default password shown
-      in :ref:`ig-default-passwords`
-   b. Follow the prompts to complete the password change
+      **Please make sure that you change these passwords!**
 
-      * See the :ref:`faq-password-complexity` FAQ for tips on setting a
-        functional password.
+  a. Change the ``root`` user password.
 
-#. Change the ``simp`` user password
+    i.  At the console, log on as ``root`` and type the default password shown
+        in :ref:`ig-default-passwords`
+    ii. Follow the prompts to complete the password change
 
-   a. At the console, log on as ``simp`` and type the default password shown
-      in :ref:`ig-default-passwords`
-   b. Follow the prompts to complete the password change
+      .. NOTE::
 
-      * See the :ref:`faq-password-complexity` FAQ for tips on setting a
-        functional password.
+         See the :ref:`faq-password-complexity` FAQ for tips on setting a
```

Comment:
Seems like this NOTE:  should be listed once before 'Change the root user password', instead of being repeated in 'Change the root user password' and 'Change the simp user password'

Reply:
I agree.